### PR TITLE
[tsp-client] Update rest-api-diff dependency

### PR DESCRIPTION
**Pull request by tjprescott**

### CODE REVIEW

**Review by mikeharder — APPROVED** [review]

File: tools/tsp-client/package.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
     "@autorest/openapi-to-typespec": "0.8.2",
     "@azure-tools/typespec-autorest": ">=0.44.0 <1.0.0",
     "@azure/core-rest-pipeline": "^1.12.0",
-    "@azure-tools/rest-api-diff": "^0.1.0",
+    "@azure-tools/rest-api-diff": ">=0.1.0, <1.0.0",
```
Comment (mikeharder) [inline tools/tsp-client/package.json:58]:
```suggestion
    "@azure-tools/rest-api-diff": ">=0.1.0 <1.0.0",
```

**Review by mikeharder — CHANGES_REQUESTED** [review]
Remove comma

**Review by mikeharder — APPROVED** [review]

**Review by catalinaperalta — APPROVED** [review]

**Pull request merged**